spec/controllers/snipets_controller_spec.rb

```diff
@@ -57,6 +58,7 @@
 
   describe "GET #new" do
     it "assigns a new snipet as @snipet" do
+      sign_in  User.new(email: 'landry@spaceinvade.rs', password: '123456', username: 'landry')
```

Comment:
Had very hard times trying to figure out how to use shared context, so in the end I just improved my code a little by using FactoryGirl for creating a snippet.
Shared contexts will definitely make a good candidate for a future workshop !